### ISSUE TITLE
[Bug]: Need ability to set tab order of widgets on a form

### ISSUE THREAD

**Issue by samiphoenix84** [appsmithorg/appsmith#36918]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Description

This issue seems to have been raised many times before but it has never been fixed. I have several text input and select widgets on a form. When I press the tab key the focus hops from one to the next in a random order. I need the ability to set the tab index of each widget. Only editable widgets should be included in the tab order, so if a widget is disabled then it should be skipped. This feature is very important to ensure forms are accessible - especially for visually impaired. Even PowerApps has this feature!

### Steps To Reproduce

Create a form, add some widgets and try to change the tab order.

### Environment

Production

### Severity

Medium (Frustrating UX)

### Issue video log

https://www.loom.com/share/27c0198a6de1446a8dffebe2a7f0a5fb?sid=704307e8-dc0d-4908-a5f3-51f5b791d0ac

### Version

Cloud

**Comment by Nikhil-Nandagopal** [comment]
Closing this as a duplicate of #19935
